src/pages/guides/api/eventsingress_api.md

```diff
@@ -22,44 +22,71 @@ Please note that all custom events are stored in the **United States (US)** regi
+
 * `api_key` is the api-key associated with your Adobe Developer Console workspace
 * `jwt_token` is a jwt token generated using the set up from the same workspace
+* `is_phidata` is a boolean value indicating if the event contains PHI data, and is required for [HIPAA compliance](#hipaa-compliance-support) (defaults to `false` if not provided)
 * `provider_id` is your `Custom Events Provider` uuid generated by [Adobe I/O Events Provider API](../api/provider_api.md)
 * `event_code` is the `Custom Events Provider`'s `Event Metadata`'s `event_code` as persisted using [Adobe I/O Events Provider API](../api/provider_api.md)
-* `event_id` is any id of your choice (examples: UUID, event counter) guaranteeing that `source + id` is unique 
+* `event_id` is any id of your choice (examples: UUID, event counter) guaranteeing that `source + id` is unique
 for each distinct event see [CloudEvents spec](https://github.com/cloudevents/spec/blob/v1.0/spec.md#id)
 *  as for the value of `data` in the CloudEvents body payload, it can be any json payload.
 
 The API returns
+
 * HTTP Status 200 (OK) if the event has been processed correctly and there are active registrations for the event,
 * HTTP Status 204 (No Content) if there are no registrations for the event,
 * HTTP Status 429 (Too Many Requests) if your api-key is being throttled (see our [Throttling Policy](#throttling-policy)).
 * as well as the usual (4xx/5xx) error codes if there was an issue in processing the request.
+
+## HIPAA Compliance Support
+
+I/O Events is compliant with [HIPAA regulation](https://www.hhs.gov/hipaa/for-professionals/privacy/index.html#:~:text=The%20HIPAA%20Privacy%20Rule%20establishes,care%20providers%20that%20conduct%20certain).
+The event producers are responsible to signal if the event contains PHI data by setting the optional `x-event-phidata` header to `true` in the request.
+When the header is present and set to `true`:
+
+* the resulting CloudEvents-formatted event which is produced will be enriched with a [CloudEvents extension](https://github.com/cloudevents/spec/blob/main/cloudevents/primer.md#cloudevent-extension-attributes), namely, a `phidata` node, set to `true`,
+* with the above `phidata` extension receivers will be able to identify which events contain PHI data and handle them accordingly,
+* audit trace logs will be preserved for 1 year in a HIPAA-compliant storage,
+* the `id` field in the CloudEvents payload will be used as a `trace-id` for the above audit trace logs.
+
+In the event of an audit, you will need to be able to provide evidences of how the patient data was handled.
+
+### Audit log: storing the mapping between PHI data to ids
+
+It is your responsibility to store the `id` in your own logs or storage solution for traceability,
+as well as maintaining a mapping between the `id` you produce and the PHI data that the message with
+that `id` contains to be able to trace back the PHI data in case of an audit.
```

Comment:
I am not quite sure what you mean here. The clients only need to store the event IDs that contain phi data right? What mapping are we talking about?

Reply:
They need to also be able to make the connection between `id` and which PHI data that event is connected.
i.e. if you (the producer) have PHI data of A and B, you need to make sure if A asks for an audit, that you ask I/O Events for all and only ids related to PHI data of A (in the time period requested by the audit)

Reply:
I think since custom event providers are registered at the org level, the same event provider will not have PHI data of A and B.

Reply:
Can't an organization handle multiple patients data?

Reply:
I refactored the section for clarity